.nvmrc

```diff
@@ -0,0 +1 @@
+v10.14.2
```

Comment:
With an `.nvmrc` anybody using nvm will know they're using the right version of Node for local dev.

I picked node@10 since that was the minimum in `engines`.